frontend/src/pages/groups/GroupCreatePage.tsx

```diff
@@ -0,0 +1,66 @@
+import { FunctionComponent } from 'react'
+import LayoutWithNav from '../../layouts/LayoutWithNav'
+import { Group, GroupInput } from '../../types/api-types'
+import { gql, useMutation } from '@apollo/client'
+import { useHistory } from 'react-router-dom'
+import GroupForm from './GroupForm'
+
+const ADD_GROUP = gql`
+  mutation Groups($input: GroupInput!) {
+    addGroup(input: $input) {
+      id
+    }
+  }
```

Comment:
Suggestion (non-blocking): it'd be nice to pull these into their own files e.g. https://github.com/distributeaid/toolbox/blob/saga/react_ui/src/pages/createChapter.graphql

Reply:
I like that a lot, thank you for the suggestion. I've opened #69 to keep track of this.